[torch] add use_buffers to swa_utils interface

galrotem opened the pull request:
Summary: As title, this already exists in swa_utils.py

Differential Revision: D49155243

janeyx99 commented:
@galrotem you're going to need to sign the CLA or add your meta email to your git account

galrotem commented:
/easycla

galrotem committed:
[torch] add use_buffers to swa_utils interface (#109078)

Summary:
X-link: https://github.com/pytorch/tnt/pull/541


As title, this already exists in swa_utils.py

Internal

I saw this when we wanted to add type annotations in TorchTNT (see next diff). The param already exists in the `.py` file so I guess we can add it here as well

Test Plan: sandcastle

Differential Revision: D49155243
